python/cudf/cudf/utils/ioutils.py

```diff
@@ -473,8 +473,36 @@
 engine : {{ 'auto', 'cudf', 'cudf_experimental', 'pandas' }}, default 'auto'
     Parser engine to use. If 'auto' is passed, the engine will be
     automatically selected based on the other parameters.
-orient : string,
-    Indication of expected JSON string format (pandas engine only).
+lines : boolean, default False
+    Read the file as a json object per line.
+dtype : boolean or dict, default True
+    If True, infer dtypes, if a dict of column to dtype, then use those,
+    if False, then don't infer dtypes at all, applies only to the data.
+byte_range : list or tuple, default None (cudf engine only)
+    Byte range within the input file to be read.
+    The first number is the offset in bytes, the second number is the range
+    size in bytes. Set the size to zero to read all data after the offset
+    location. Reads the row that starts before or at the end of the range,
+    even if it ends after the end of the range.
+keep_quotes : bool, default False (cudf_experimental engine only)
+    If `True`, any string values are read literally (and wrapped in an
+    additional set of quotes).
+    If `False` string values are parsed into Python strings.
+typ : type of object to recover (series or frame), default 'frame'
+    With cudf engine, only frame output is supported.
+encoding : str, default is 'utf-8'
+    The encoding to use to decode py3 bytes.
```

Comment:
What are `py3 bytes` ?

Reply:
It a pandas.read_json only param, hence we just copied it here.